RestaurantReservation.Domain/AppMetaData/Router.cs

```diff
@@ -0,0 +1,87 @@
+﻿namespace RestaurantReservation.Domain.AppMetaData;
+public static class Router
```

Comment:
as I said to you, I hate this pattern. You must avoid declaring the routing as this, no problem in hard writing them in their place